### DISCUSSION TITLE
122/5000 milvus Precision storage and display

### DISCUSSION THREAD

**Discussion by sulv9999** [milvus-io/milvus#36529] [Q&A and General discussion]
Hello, I have a question. milvus precision storage is not consistent with the display and the actual observation

In the auut client, I see that the stored vector feature is double-floating-point precision, but the official documentation states that milvus only supports Float32, why is it double-floating-point when I look at mivlus

![image](https://github.com/user-attachments/assets/8225fd84-34c6-4152-8c1a-c1d1e574d1d7)

**Comment by yhmo** [comment]
Read this discussion to get the answer: https://github.com/milvus-io/milvus/discussions/32737
